scripts/drone/pipelines/aws_marketplace.star

```diff
@@ -0,0 +1,38 @@
+load(
+    'scripts/drone/steps/lib.star',
+    'download_grabpl_step',
+    'publish_images_step',
+    'compile_build_cmd',
+    'fetch_images_step',
+    'publish_image',
+)
+
+load('scripts/drone/vault.star', 'from_secret')
+
+load(
+    'scripts/drone/utils/utils.star',
+    'pipeline',
+)
+
+def publish_aws_marketplace_step():
+    return {
+        'name': 'publish-aws-marketplace',
+        'image': publish_image,
+        'commands': ['./bin/build publish aws --image grafana/grafana-enterprise --repo grafana-labs/grafana-enterprise --product 422b46fb-bea6-4f27-8bcc-832117bd627e'],
```

Comment:
Since the AWS region is configured externally, would it perhaps also make sense to move the product ID also into the Vault?

Reply:
@guicaulada said it's okay for this to be exposed, that's why I left it there.
No big deal to move it though.

Reply:
The Product ID is visible on the product page after subscribing to the product, so it is a public information that is already shared through AWS.